gxff/gxff.go

```diff
@@ -121,7 +123,7 @@ func parseTrustedProxies() []string {
 // If allowedSubnets is nil and TRUSTED_PROXIES is empty then X-FORWARDED-FOR will be ignored (no proxy is trusted).
 func New(l log.Logger, allowedSubnets []string) (grpc.StreamServerInterceptor, grpc.UnaryServerInterceptor) {
```

Comment:
We should rename this package to just `xff`, and change `New` to `GRPCMiddleware` and leave it to the caller to pass in a trusted proxies or not.